lib/after_commit_everywhere.rb

```diff
@@ -120,6 +125,10 @@ def register_callback(connection: nil, name:, without_tx:, callback:)
       connection ||= default_connection
       wrap = Wrap.new(connection: connection, "#{name}": callback)
       connection.add_transaction_record(wrap)
+      if prepend
+        records = connection.current_transaction.instance_variable_get(:@records)
+        records.unshift(records.pop)
```

Comment:
Is there a reason why we need to add `wrap` to records twice: once at the end, then remove, and re-add to the head?

Something like this should be more readable:
```ruby
if prepend
  # …
else
  connection.add_transaction_record(wrap)
end
```

Reply:
Fair point ✅